experimental/generation/generator/packages/library/templates/standard/en-us/dynamiclistProperty.en-us.lg.lg

```diff
@@ -0,0 +1,43 @@
+[standard.en-us.lg](standard.en-us.lg)
+
+# filename
+- ${propertyFilename()}
+
+# template
+- ```
+${formImport('form.en-us.lg')}
+${importLG(`${entity}`)}
+${promptTemplates(missingDefault(), changeDefault())}
+${confirmationTemplates(expectedSetDefault(), 
+                        expectedUpdateDefault(),
+                        unexpectedSetDefault(),
+                        unexpectedUpdateDefault(),
+                        removeDefault(),
+                        clearDefault())}
+${helpTemplates(repromptEnum(), helpEnum())}
+${showTemplate(showDefault())}
+${nameTemplate()}
+${enumValueTemplate()}
+```
+
+# repromptEnum()
+- IF: ${isArrayProperty(property)}
+    - ${propertyName()} includes multiple values from: \${possibleValuesList(`${property}`)}
```

Comment:
>possibleValuesList [](start = 58, length = 18)

I don't think you can use this--the possible values is defined dynamically not at generation time.